tests/python/contrib/test_tensorrt.py

```diff
@@ -1034,5 +1038,186 @@ def set_func_attr(func, compile_name, symbol_name):
     tvm.ir.assert_structural_equal(mod_trt, mod_exp, map_free_vars=True)
 
 
+def test_tensorrt_dynamic_batch():
+    if skip_codegen_test():
+        return
+
+    batches_to_test = [1, 1, 2, 3, 1, 3, 2]
```

Comment:
@trevor-m Should we test batch size of 0 here?

Reply:
Done.